pybess/bess.py

```diff
@@ -48,15 +48,18 @@
 # Ugh: builtin_pb and plugin_pb must be on path, as protoc generates python code
 # that assumes it can import files in that directory.
 old_path = list(sys.path)
+p = os.path.abspath(os.path.dirname(__file__))
+if p not in sys.path:
+    sys.path.append(p)
```

Comment:
You might want to `insert` at 1 here for efficiency-and-override effect (same as in the loop below), or else change all of them to use `append` - it seems a bit weird/inconsistent now.

Reply:
Ah-- they probably all should be an `append`, otherwise things a user adds to PYTHONPATH will come after the `insert(1, p)`. This would mean that PYTHONPATH won't properly override. For my specific use-case, I actually only need to override the first one (L53), but we should do all of them for consistency.